tests/st/policy/test_profile.py

```diff
@@ -30,15 +30,16 @@
 class MultiHostMainline(TestBase):
     @parameterized.expand([
         #"tags",
-        "rules.tags",
+        ("rules.tags", False),
+        ("rules.tags", True),
```

Comment:
Once GoBGP supports IPIP fully I assume we should be running these tests on GoBGP?

No action for you here.  Just wondering whether GoBGP might be affected by the same issue as Bird.